src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+        Storage.people.add(new User("Jane","123dsafq",18));
+        Storage.people.add(new User("Natia","addarw41", 30));
+        Storage.people.add(new User("Thomas","djfj3621", 20));
+    }
+
+    @Test
+    void register_userWithUniqueLogic_Ok() throws ValidationExeption {
+        user = new User("Frank","82ghnrw3",35);
+        registrationService.register(new User("Frank","82ghnrw3",35));
+        assertEquals(user, storageDao.get("Frank"));
+    }
+
+    @Test
+    void register_isLoginInStorage_NotOk() {
+        user = new User("Jane","82ghnd43",31);
+        assertThrows(ValidationExeption.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_loginIsNull_NotOk() {
+        user = new User(null,"82ghnd43", 31);
+        assertThrows(ValidationExeption.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_loginIsEmptyLine_NotOk() {
+        user = new User("","82ghnd43", 31);
+        assertThrows(ValidationExeption.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_userAgeIsEighteen_Ok() throws ValidationExeption {
+        user = new User("Forty","87lhnd43", 18);
+        registrationService.register(user);
+        assertEquals(user, storageDao.get("Forty"));
+    }
+
+    @Test
+    void register_userAgeIsGreaterThanEighteen_Ok() throws ValidationExeption {
+        user = new User("Yesmin","82ghfd43", 91);
+        registrationService.register(user);
+        assertEquals(user, storageDao.get("Yesmin"));
+    }
+
+    @Test
+    void register_userAgeIsNull_NotOk() {
+        user = new User("Marta","fortycash", null);
+        assertThrows(ValidationExeption.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_userAgeLessThanEighteen_NotOk() {
+        user = new User("Marta","fortycash1", 17);
+        assertThrows(ValidationExeption.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_userPasswordIsSixCharacters_Ok() throws ValidationExeption {
+        user = new User("Kobe","ghfd43", 20);
+        registrationService.register(user);
+        assertEquals(user, storageDao.get("Kobe"));
+    }
+
+    @Test
+    void register_userPasswordIsGreaterThanSixCharacters_Ok() throws ValidationExeption {
+        user = new User("George","ghfd4382daa", 19);
+        registrationService.register(user);
+        assertEquals(user, storageDao.get("George"));
+    }
+
+    @Test
+    void register_userPasswordIsNull_NotOk() {
+        user = new User("Monika", null, 30);
+        assertThrows(ValidationExeption.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_userPasswordIsEmptyLine_NotOk() {
+        user = new User("Stephen", "", 34);
+        assertThrows(ValidationExeption.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_userPasswordIsLessThanSixCharacters_NotOK() {
```

Comment:
dont put concrete values in method names
```suggestion
    void register_userPasswordIsLessThanMinRequired_NotOK() {
```

Reply:
done